### DISCUSSION TITLE
Setting up Frigate to use external storage in HA

### DISCUSSION THREAD

**Discussion by VagnPetersen** [blakeblackshear/frigate#8074] [Ask A Question] [answered]
I am trying out Frigate as an addon on my HA setup (Odroid N2+). I have a simple one camera setup and everything works fine.
I am planning on getting a Coral TPU shortly and are just checking out the possibilities of Frigate.
Right now I have problems understanding how "HA network storage" is suppose to work. I have no problems following the online instructions and setting up an external storage on my NAS (mounted as "Frigate" in the root of my NAS).  Two folders are created (Clips and Recordings) and all detected events and recordings are present.
However, I also have the same files on my Odroid (/media/frigate) and it is simply a mirror of what are on my NAS (or vise versa).
If I stop the addon and manually deletes the frigate folder, it re appers when I start the addon and I can se from the logfile, that the startup script creates the folder structure on the Odroid running HA.
Is it possible to have Frigate to only save medias in one location?

**Comment by NickM-27** [comment, marked as answer]
> However, I also have the same files on my Odroid (/media/frigate) and it is simply a mirror of what are on my NAS (or vise versa).
If I stop the addon and manually deletes the frigate folder, it re appers when I start the addon and I can se from the logfile, that the startup script creates the folder structure on the Odroid running HA.
> Is it possible to have Frigate to only save medias in one location?

You are misunderstanding how this is working. The folder on the HA side is simply a mount of the NAS volume, the files are not duplicated on the main devices storage. Other users have found this confusing the way it is presented but of course we have no control over this.

https://community.home-assistant.io/t/frigate-records-on-nas-and-home-assistant/617078/5

**Reply by VagnPetersen** [reply]
OMG! You are right.
I tried to delete a few files on my NAS. And yes - it is mirrored on my HA installation,
I didn't realise this. Thank you SO MUCH!
